pom.xml

```diff
@@ -219,6 +219,13 @@
     </build>
 
     <repositories>
+        <repository>
```

Comment:
so we're just not using this at all? 🤔

Reply:
Ah, so this just explicitly adds a repository of images for maven to look in when it's searching for the geotools dependency, because the original location where they had been publishing that dependency changed, causing this error in our build:

"Failed to execute goal on project graphhopper-web-bundle: Could not resolve dependencies for project com.graphhopper:graphhopper-web-bundle:jar:0.13-SNAPSHOT: The following artifacts could not be resolved: org.geotools:gt-epsg-hsql:jar:19.0, org.geotools:gt-metadata:jar:19.0, org.geotools:gt-api:jar:19.0, org.geotools:gt-shapefile:jar:19.0: Could not find artifact org.geotools:gt-epsg-hsql:jar:19.0 in graphhopper-matrix"

https://circleci.com/gh/replicahq/graphhopper/81?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

It _says_ that it can't find it in "graphhopper matrix", but that's just because maven looks at repos provided in order, and fails if it can't find the dep in the last one it checks (I think)

Reply:
And you can see that the build passes now, woohoo!

Reply:
got it, cool lgtm!